index.bs

```diff
@@ -892,8 +883,12 @@ user consent to a specific transaction. The structure of these signatures is def
 
 ## Additional options for Assertion Generation (dictionary <dfn dictionary>AssertionOptions</dfn>) ## {#assertion-options}
```

Comment:
Editorial: Could we remove the word "additional here, since this is now all the options?